docs/open-roles/cloud-service-certification-project-maintainer.md

```diff
@@ -0,0 +1,36 @@
+# CSC Project Maintainer - Cloud Service Certification 
+Helping run [Cloud Service Certification](https://github.com/finos/cloud-service-certification) with the Cloud Service Certification Lead Maintainer as CSC Project Maintainer is a fun open source role involving a broad set of tasks that need to be taken care of, including ...
+
+## Role Requirements
+
+- You're an experienced senior level cloud services engineer within the financial services industry and understand the relationship between cloud engineering, finacial services regulation, cyber security, risk and banking policies within an enterprise banking setting.
+
+- You'll review and provide feedback on pull requests. The CSC community is growing fast and we're receiving a growing number of pull requests. You will help review pull requests, provide constructive feedback and support contributors, and ultimately making the call on what is merged into the project.
+
+- You'll help guide architectural decisions. The future of Cloud Service Certification is discussed and planned on project calls and in our GitHub Issues. You’ll help drive these conversations and will guide CSC towards the best possible solutions.
+
+- You'll convert high-level ideas into detailed specifications and actionable issues. You will provide the required context to allow community contributors to quickly and effectively pick up issues.
```

Comment:
Given that we are asking the sub teams to do most of this now, can we change this wording?  Perhaps to something like 'You will contribute to the detailed solutions and support the rest of the community in building out the features.'

Reply:
@peter-thomas-db

This is addressed in the commit below below ...

- update @peter-thomas-db role feedback d86df20

James.